wgpu-hal/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "wgpu-hal"
+version = "0.1.0"
+authors = ["wgpu developers"]
+edition = "2018"
+description = "WebGPU hardware abstraction layer"
+homepage = "https://github.com/gfx-rs/wgpu"
+repository = "https://github.com/gfx-rs/wgpu"
+keywords = ["graphics"]
+license = "MPL-2.0"
```

Comment:
Can it be `MIT/Apache-2.0` since it's a new crate?

Reply:
good point, fixed